### ISSUE TITLE
ABI checker should provide computer readable format

### ISSUE THREAD

**Issue by phracek** [rebase-helper/rebase-helper#113]
In order to parse output from ABI Checker it would be good to have computer readable format.

Some references are already here: https://github.com/phracek/rebase-helper/issues/101

Several links:
https://sourceware.org/git/gitweb.cgi?p=libabigail.git;a=blob_plain;f=doc/manuals/ifac-schema.rng;hb=refs/heads/xml-report
https://sourceware.org/git/gitweb.cgi?p=libabigail.git;a=shortlog;h=refs/heads/xml-report

**Comment by dodjiseketeli** [comment]
Yeah, I guess have to revive (and finish) that 'xml-report' branch from the libabigail code repository.

**Comment by dodjiseketeli** [comment]
I have opened a tracker bug for the libabigail project specifically at https://sourceware.org/bugzilla/show_bug.cgi?id=18716, to add support for a well defined XML format for the ABI change report emitted by abipkgdiff (and abidiff).

If you need anything specific related to this request, please feel free comment on that bug there.

**Comment by phracek** [comment]
Because of this issues depends on libabigail implementation and bug is still in NEW, moving to 0.9.0 milestone.

**Comment by phracek** [comment]
@dodjiseketeli Any update? https://sourceware.org/bugzilla/show_bug.cgi?id=18716

**Comment by dodjiseketeli** [comment]
Sadly, I haven't had time to work on this yet.  It's still something I definitely plan to work on, though.  It's just that I have been busy on other more pressing aspects of libabigail.  I hope to get to this at some point, unless someone gets to it first.

**Comment by phracek** [comment]
Closing this issue. I do not work on this project. It is outdated. Feel free to re-open it.
